pipelines/visitQAEllipPlots.yaml

```diff
@@ -0,0 +1,364 @@
+description: Make visit level ellipticity plots for QA
```

Comment:
A slightly extended description would be wonderful here (and for the other yaml), like if you run this on a visit, you will get 20 plots and they will be saved as gifs and automatically tweeted.